src/forms/FormFieldColorPicker/FormFieldColorPicker.tsx

```diff
@@ -28,7 +47,7 @@ const FormFieldColorPicker = (
 	
 	const onColorChange = (color: ColorResult) => {
 		setColor(color);
-		onChange(color.hex);
+		onChange(RGBAToHexA(color.rgb));

```

Comment:
This is good for changing the color from the field to the form, but we should also have a mechanic (probably in a useEffect) that processes the value when it goes from the form into the field. In this example I opened the Form Playground, added the value to the color picker: "#a8001791", clicked prepopulate, and this is what it looked like: 
![image](https://user-images.githubusercontent.com/23409284/183769530-d3c6ecf9-b2f9-488e-9bed-9b08c59d8566.png)